website/templates/public/pages/getting_started.mako

```diff
@@ -65,7 +65,7 @@
         </div>
     <div class="col-sm-9">
 
-        <div  id="start"  class="col-sm-12">
+        <div  id="start"  class="col-sm-12" style="margin-left: 40px">
```

Comment:
@sloria Agreed. I went ahead and changed this. 
Gotta merge with develop first to see the changes since I didn't originally create a PR comparing the fix branch but rather compared my develop after merging the fix branch. Was a mistake.